app/Car.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+

```

Comment:
te nav norādīts namespace, līdz ar to programma visticamāk šobrīd nestrādā pareizi